serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -470,11 +419,7 @@ To tell {product_name} you want to send the payload as a CloudEvent, you must de
     {
       "name": "greet",
       "type": "custom",
-      "operation": "knative:custom-function-knative-service",
-      "metadata": {
-        "path": "/",
-        "asCloudEvent": true <1>
-      }
+      "operation": "knative:services.v1.serving.knative.dev/custom-function-knative-service?path=/plainJsonFunction&asCloudEvent=true" <1>
```

Comment:
Isn't the equivalent here the `/` path? Why is there `/plainJsonFunction` everywhere?